src/toil_scripts/spark_utils/spawn_cluster.py

```diff
@@ -6,44 +6,44 @@
 """
 
 import logging
+import multiprocessing
+import os
+from subprocess import call, check_call, check_output
+import time
 
 from toil.job import Job
 from toil_scripts.batch_alignment.bwa_alignment import docker_call
 
-log = logging.getLogger(__name__)
+_log = logging.getLogger(__name__)
```

Comment:
It's related to @hannes-ucsc's earlier comment RE: not `from <module> import *`ing unless `__all__` is defined. TL;DR is that prefixing a field/method with `_` is a Python convention for something being private. It's more a convention than a hard access control, though.